--- a/file.c
+++ b/file.c
@@ -22,6 +22,7 @@
 #include <sys/queue.h>
 #include <sys/socket.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <sys/uio.h>
 #include <sys/wait.h>
 #ifdef __linux

 
 #include <err.h>
 #include <errno.h>
--- a/seccomp-sandbox.c
+++ b/seccomp-sandbox.c
@@ -109,6 +109,9 @@ static const struct sock_filter filt_insns[] = {
 #ifdef __NR_write
 	SC_ALLOW(__NR_write),
 #endif
+#ifdef __NR_writev
+	SC_ALLOW(__NR_writev),
+#endif
 
 	/* Default deny. */
 	BPF_STMT(BPF_RET+BPF_K, SECCOMP_RET_KILL),
